docs/note.md

```diff
@@ -47,9 +47,9 @@ The following is an example of the <file>secret-share.md</file> alert added usin
  <p>It renders <i>vanilla</i> HTML and markdown, Alerts, and images. For example, these two images:</p>
 
 - **Markdown Image Example**<br>
-![expand example](/img/expander-markdown.png)<br>
+![expand example](/general/expander-markdown.png)<br>
 - **HTML Image Example*- (with border)<br>
-<img style="border:solid 1px black" src="/img/expander-markdown.png" alt="Screen capture of Tab/Tabs Shortcode Usage">
+![Screen capture of Tab/Tabs Shortcode Usage](/general/expander-markdown.png)
```

Comment:
{{<imgproc src="/general/expander-markdown.png" resize="800x" declaredimensions=true alt="Screen capture of Tab/Tabs Shortcode Usage">}}